lib/project-data.ts

```diff
@@ -103,10 +107,17 @@ export class ProjectData implements IProjectData {
 				const appResourcesDirPath = configNS[constants.CONFIG_NS_APP_RESOURCES_ENTRY];
 
 				absoluteAppResourcesDirPath = path.resolve(projectDir, appResourcesDirPath);
+
+				if (this.$fs.exists(absoluteAppResourcesDirPath)) {
```

Comment:
Check what happens if we go throu here https://github.com/NativeScript/nativescript-cli/blob/27e937b71f27234a62d02c3ace7d1896aaa7d05e/lib/services/project-service.ts#L113

Reply:
@KristianDD 
Creating an ng/ts project, which would suggest that 2 templates will be downloaded - one before `ensureAppResourcesExist` is called, the other one - afterwards, doesn't make the process crash. I am not sure if the case would be the same if the template is shipped with nsconfig.json which points to App_Resources.

What do you think? Should I perhaps return the path, even if the directory doesn't exist, as that's where the App_Resources from the template should be extracted?